docs/trace/getting-started/Program.cs

```diff
@@ -36,5 +36,6 @@ public static void Main()
         activity?.SetTag("foo", 1);
         activity?.SetTag("bar", "Hello, World!");
         activity?.SetTag("baz", new int[] { 1, 2, 3 });
+        activity?.SetStatus(ActivityStatusCode.Ok);
```

Comment:
depending on other's opinion on this, I will explore documenting this somewhere else...

Reply:
maybe Error is better for demo purpose?

Reply:
I thought about it.. But it'd be more apt to show Error, with an Exception.. that would then bring all the complexities associated with Exception into a getting-started doc..... (we already have a section on Exception-reporting, which I'll tweak to reflect new Status api)

Reply:
Makes sense, thanks!